src/bp-forums/common/shortcodes.php

```diff
@@ -226,7 +226,7 @@ public function display_forum( $attr, $content = '' ) {
 			// Sanity check required info.
 			if ( ! empty( $content ) || ( empty( $attr['id'] ) || ! is_numeric( $attr['id'] ) ) ) {
 				return $content;
-			}
+			}	
```

Comment:
@kent-carlo These changes in not required. Can you verify changes before the final commit?